src/sentry/models/release.py

```diff
@@ -458,7 +459,7 @@ class Release(Model):
     A commit is generally a git commit. See also releasecommit.py
     """
 
-    __include_in_export__ = False
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
I bet we'll need to add this and a handful of other models to the Organization scope before relocation work is complete.